paywall/src/paywall-builder/config.js

```diff
@@ -26,6 +48,15 @@ export function setupReadyListener(window, iframe, origin) {
       // <script type="text/javascript">window.unlockConfig = {...}</script>
       // immediately before paywall.min.js is loaded
       sendConfig(window.unlockConfig, iframe, origin)
+      if (window.web3 && window.web3.currentProvider) {
```

Comment:
the logic inside `getAccount` needs to be called twice, so the result of moving it inside getAccount would be to repeat the `send` call twice. I actually extracted `getAccount` because of this need.  However, I *can* make a function that either waits for enable or executes a callback right away, which would be cleaner